source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidHelperFragment.cs

```diff
@@ -207,6 +205,42 @@ public static void ShowLeaderboardUI(string leaderboardId, LeaderboardTimeSpan t
             }
         }
 
+        public enum WaitingRoomUIStatus
+        {
+            Valid = 1,
+            Cancelled = 2,
+            LeftRoom = 3,
+            InvalidRoom = 4,
+            Busy = -1,
+            InternalError = -2,
+        }
+
+        // first callback param type will be enumerated.
```

Comment:
Not sure about this comment - why do we need it? Isn't it already clear from Action type?

Reply:
Agree, this is redundant. Removed.